app/server/models/token.py

```diff
@@ -1,9 +1,15 @@
 from server import db
 from server.utils.blockchain_tasks import (
-    get_token_decimals
+    get_token_decimals,
 )
-from server.models.utils import ModelBase
 
+from server.models.utils import (
+    ModelBase,
+    exchange_contract_token_association_table
+)
+
+from server.models.exchange import ExchangeContract
+import server.models.transfer_account
```

Comment:
circular dependency or why do it this way?

Reply:
yeah circular dep. `import server.models.transfer_account` style imports didn't even consistently work, so there was a bunch of trial and error :|